test/Interop/Cxx/operators/Inputs/member-inline.h

```diff
@@ -16,6 +16,10 @@ struct LoadableIntWrapper {
   int operator()(int x, int y) {
     return value + x * y;
   }
+
+  int operator~() {
+    return ~value;
+  }
 };
 
 struct AddressOnlyIntWrapper {
```

Comment:
Do you mind adding an out-of-line (non member) test and an address only int wrapper test?

Reply:
Sure, and I'll refrain on operator!.